wolfSSL/src/chapter02.md

```diff
@@ -813,12 +1697,37 @@ extern time_t m2mb_xtime_ms(time_t * timer);
 Define this to check for possible glitching attack against the AES encrypt
 operation during a TLS connection.
 
+#### RTTHREAD
+
+RT-THREAD macro is used when porting rtthread IoT to wolfSSL.
+
+#### SO_REUSEPORT
+
+Allows local address and port reuse.
+
+#### INTIME_RTOS
+
+Port setting for INtime RTOS.
+
+#### WOLFSSL_SGX
+
+Used when porting to SGX.
+
+
 ### Reducing Memory or Code Usage
 
 #### TFM_TIMING_RESISTANT
 
 Can be defined when using fast math ([`USE_FAST_MATH`](#use_fast_math)) on systems with a small stack size. This will get rid of the large static arrays.
 
+#### ECC_TIMING_RESISTANT
+
+This is used as a Timing Resistor for ECC. It reduces heap usage but also makes it slower, hardening the curve.
```

Comment:
"Timing Resistance" not Resistor. This enables code in ecc.c to prevent side channel and differential power analysis (DPA) attacks.